app/src/models/services/Student/Student.js

```diff
@@ -105,12 +96,17 @@ class Student {
     }
   }
 
+  static idOrEmailInputNullCheck(client) {
```

Comment:
```suggestion
  static idOrEmailNullCheck(client) {
```
id랑 email이라는 키워드가 들어가있어서 input이라는 말은 필요 없을거같아요